src/component/Footer/Footer.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import cn from 'classnames';
+
+type Props = {
+  hasCompletedTodo?: boolean;
+  itemLeft: number;
+  stateTodo: boolean | undefined;
+  setStateTodo: (state: boolean | undefined) => void;
+  clearComplete: () => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  itemLeft,
+  stateTodo,
+  hasCompletedTodo = false,
+  setStateTodo,
+  clearComplete,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {itemLeft} items left
+      </span>
+
+      {/* Active link should have the 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={cn('filter__link', {
+            selected: stateTodo === undefined,
+          })}
+          data-cy="FilterLinkAll"
+          onClick={() => setStateTodo(undefined)}
```

Comment:
Use enum ('All', 'Active', 'Completed') for this function `setStateTodo` instead of undefined, false or true. It will be better